stdlib/public/Synchronization/Mutex/WindowsImpl.swift

```diff
@@ -10,7 +10,30 @@
 //
 //===----------------------------------------------------------------------===//
 
-import WinSDK.core.synch
+#if _pointerBitWidth(_64)
+@usableFromInline
+typealias SRWLOCK = UInt64
+#elseif _pointerBitWidth(_32)
+@usableFromInline
+typealias SRWLOCK = UInt32
+#else
+#error("Unsupported platform")
+#endif
+
+@usableFromInline
+typealias PSRWLOCK = UnsafeMutablePointer<SRWLOCK>
+
+@usableFromInline
+@_extern(c, "AcquireSRWLockExclusive")
```

Comment:
Can we double check the calling convention? They may be `__stdcall`.

Reply:
It seems on x86 they are stdcall like all other Win32 API, but on archs other than x86 windows just uses the C calling convention if I'm reading this correctly. I can update the PR.

Reply:
I think we may need a way to say `@_extern(stdcall)` though if we want to support x86.